app/scripts/directives/nav.js

```diff
@@ -273,10 +294,54 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        contextSelector.append($('<option value="1">Service Catalog</option>'));
+        contextSelector.append($('<option value="2">Application Console</option>'));
+        if (window.OPENSHIFT_CONSTANTS.TECTONIC_URL) {
+          contextSelector.append($('<option value="3">Cluster Console</option>'));
+        }
+        contextSelector.append($('<option data-divider="true"></option>'));
+        contextSelector.append($('<option value="4">Set Home Page</option>'));
```

Comment:
We might leave this out for the initial PR until we have the home page dialog updated to include the cluster console homepage option.